<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="edAlfa">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>301</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edSigStep">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSigmoid">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>3</width>
      <height>3</height>
     </size>
    </property>
    <property name="text">
     <string>Show sigmoid</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="img1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>220</width>
      <height>110</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbl1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SigStep</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btLern">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>lern</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Goal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lb1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>551</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>346</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="edN1">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edN2">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edN3">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="edOut1">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edOut2">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edOut3">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="edGoal1">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edGoal2">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edGoal3">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>init wt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>tmp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>tmp1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>tmp2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>test mx</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>edAlfa</zorder>
   <zorder>edSigStep</zorder>
   <zorder>btSigmoid</zorder>
   <zorder>img1</zorder>
   <zorder>lbl1</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>btLern</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>lb1</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
